File: Fronius/Models/PhotovoltaicSystem.cs
using Fronius.Dtos;

namespace Fronius.Models {
    public class PhotovoltaicSystem {
        public PhotovoltaicSystem(PhotovoltaicSystemDetailDto pv) {
            Id = pv.Id;
            Name = pv.Name;
            PeakPower = pv.PeakPower;
            EnergyProducedCurrentDay = pv.EnergyProducedCurrentDay;
        }

        public Guid Id { get; set; }

        public string Name { get; set; }

        public double PeakPower { get; set; }

        public double EnergyProducedCurrentDay { get; set; }
    }
}

File: Fronius/ClientApp/src/app/models/photovoltaic-system.ts
export interface PhotovoltaicSystem {
  id: string;
  name: string;
  peakPower: number;
  energyProducedCurrentDay: number;
}

export interface PhotovoltaicSystemDetail {
  id: string;
  name: string;
  peakPower: number;
  energyProducedCurrentDay: number;
  installationDate: Date;
  owner: string;
  address: Address;
  systemCondition: Condition;
}

export interface Address {
  latitude: number;
  longitude: number;
  countryCode: string;
  countryName: string;
  street: string;
  zipCode: string;
}

export enum Condition {
  Good = 0,
  Warning = 1,
  Error = 2
}

